packages/vhd-lib/src/Vhd/VhdAbstract.js

```diff
@@ -37,8 +29,18 @@ export class VhdAbstract {
     return computeSectorsPerBlock(this.header.blockSize)
   }
 
-  set header(header) {
-    this.#header = header
+  get header() {
+    throw new Error('get header is not implemented')
+  }
+  set header(_) {
+    throw new Error('set header is not implemented')
+  }
+
+  set footer(_) {
+    throw new Error('set footer not implemented')
+  }
+  get footer() {
+    throw new Error('get footer not implemented')
```

Comment:
Small remark: only defining getters is enough as the properties will not be settable by default:

```
o.foo = bar
      ^

TypeError: Cannot set property foo of #<Object> which has only a getter
```

Reply:
nice, didn't know this one